--- translate-toolkit-2.1.0/translate/tools/poterminology.py~	2017-04-23 18:23:57.000000000 +0300
+++ translate-toolkit-2.1.0/translate/tools/poterminology.py	2017-04-23 18:24:31.871529381 +0300
@@ -79,7 +79,7 @@
 
         if stopfile is None:
             try:
-                stopfile = file_discovery.get_abs_data_filename('stoplist-%s' % self.sourcelanguage)
+                stopfile = '/usr/share/translate-toolkit/stoplist-%s' % self.sourcelanguage
             except Exception:
                 pass
         self.stopfile = stopfile
@@ -457,7 +457,7 @@ def main():
 
     parser.add_option("-S", "--stopword-list", type="string", metavar="STOPFILE", dest="stopfile",
                       help="read stopword (term exclusion) list from STOPFILE (default %s)" %
-                      file_discovery.get_abs_data_filename('stoplist-en'))
+                      '/usr/share/translate-toolkit/stoplist-en')
 
     parser.set_defaults(foldtitle = True, ignorecase = False)
     parser.add_option("-F", "--fold-titlecase", callback=fold_case_option,
