diff -up translate-toolkit-1.5.3/translate/tools/poterminology.py.stoplist translate-toolkit-1.5.3/translate/tools/poterminology.py
--- translate-toolkit-1.5.3/translate/tools/poterminology.py.stoplist	2010-01-23 01:18:39.000000000 +0200
+++ translate-toolkit-1.5.3/translate/tools/poterminology.py	2010-02-02 15:35:24.000000000 +0200
@@ -74,7 +74,7 @@ class TerminologyExtractor(object):
 
         if stopfile is None:
             try:
-                stopfile = file_discovery.get_abs_data_filename('stoplist-%s' % self.sourcelanguage)
+                stopfile = '/usr/share/translate-toolkit/stoplist-%s' % self.sourcelanguage
             except:
                 pass
         self.stopfile = stopfile
@@ -457,7 +457,7 @@ def main():
 
     parser.add_option("-S", "--stopword-list", type="string", metavar="STOPFILE", dest="stopfile",
                       help="read stopword (term exclusion) list from STOPFILE (default %s)" %
-                      file_discovery.get_abs_data_filename('stoplist-en'))
+                      '/usr/share/translate-toolkit/stoplist-en')
 
     parser.set_defaults(foldtitle = True, ignorecase = False)
     parser.add_option("-F", "--fold-titlecase", callback=fold_case_option,
