diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/translate/lang/identify.py translate-toolkit-2.2.4/translate/lang/identify.py
--- translate-toolkit-2.2.4.org/translate/lang/identify.py	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/translate/lang/identify.py	2021-03-02 23:26:49.747045758 +0100
@@ -32,7 +32,7 @@ from translate.storage.base import Trans
 
 
 class LanguageIdentifier(object):
-    MODEL_DIR = get_abs_data_filename('langmodels')
+    MODEL_DIR = '/usr/share/translate-toolkit/langmodels'
     """The directory containing the ngram language model files."""
     CONF_FILE = 'fpdb.conf'
     """
