diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/junitmsgfmt translate-toolkit-2.2.4/tools/junitmsgfmt
--- translate-toolkit-2.2.4.org/tools/junitmsgfmt	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/junitmsgfmt	2021-03-02 23:26:50.197044243 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Copyright 2010 Zuza Software Foundation
 #
@@ -25,38 +25,38 @@ body=""
 failures=0
 successes=0
 
-function failure {
+failure() {
 	pofile=$1
 	body=$(echo $body; echo "<testcase classname=\"$pofile\" name=\"msgfmt\" time=\"\">\n"; echo "<failure message=\"msgfmt failure\">$message</failure>\n</testcase>\n")
 	message=""
 	failures=$(($failures + 1))
 }
 
-function success {
+success() {
 	pofile=$1
 	body=$(echo $body; echo "<testcase classname=\"$pofile\" name=\"msgfmt\" time=\"\"></testcase>\n")
 	message=""
 	successes=$(($successes + 1))
 }
 
-function run_msgfmt {
+run_msgfmt() {
 	pofile=$1
 	exit_status=$(msgfmt -cv -o /dev/null $pofile 2>/dev/null > /dev/null; echo $?)
-	message=$(msgfmt -cv -o /dev/null $pofile 2>/dev/stdout | while read i; do echo "$i\n" ; done)
+	message=$(msgfmt -cv -o /dev/null $pofile 2>/dev/stdout | while read i; do echo "$i"; echo "" ; done)
 	return $exit_status
 }
 
-function print_header {
+print_header() {
         echo "<?xml version=\"1.0\" encoding=\"utf-8\"?>"
 	echo "<testsuite name=\"\" errors=\"0\" failures=\"$failures\" skips=\"0\" tests=\"$((failures + successes))\" time=\"\" >"
 }
 
-function print_body {
+print_body() {
 	cat -
-	echo -e $body
+	echo $body
 }
 
-function print_footer {
+print_footer() {
 	cat -
 	echo "</testsuite>"
 }
diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/pocommentclean translate-toolkit-2.2.4/tools/pocommentclean
--- translate-toolkit-2.2.4.org/tools/pocommentclean	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/pocommentclean	2021-03-02 23:26:50.197044243 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Copyright 2005 Zuza Software Foundation
 #
diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/pocompendium translate-toolkit-2.2.4/tools/pocompendium
--- translate-toolkit-2.2.4.org/tools/pocompendium	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/pocompendium	2021-03-02 23:26:50.197044243 +0100
@@ -23,7 +23,7 @@
 # combined and marked fuzzy.
 
 
-function print_usage() {
+print_usage() {
 	echo "Usage: `basename $0` [options] output.po <-d directory(s)|file(s)>"
 	echo "       The first parameter is the output file, standard output if the output file is '-'."
 	echo "       Any number of directories may be specified for input files."
@@ -96,7 +96,7 @@ shift
 
 tmp_dir=`mktemp -d tmp.XXXXXXXXXX`
 
-if [ $1 == "-d" ]; then
+if [ $1 = "-d" ]; then
 	shift
 	directories=$*
 	if [ $# -eq 0 ]; then
diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/pomigrate2 translate-toolkit-2.2.4/tools/pomigrate2
--- translate-toolkit-2.2.4.org/tools/pomigrate2	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/pomigrate2	2021-03-02 23:26:50.197044243 +0100
@@ -21,7 +21,7 @@
 # pomigrate2 - migrates PO files from an old version using new POT files.
 
 
-function usage() {
+usage() {
 	echo "Usage `basename $0` [options] <product-old> <product-new> <product-templates>"
         echo
         echo "Options:"
@@ -99,7 +99,7 @@ templates=$3
 
 echo "** Migrating files... **"
 pots=`cd $templates; find . -name "*.pot"`
-if [ "$pots" == "" ]; then
+if [ "$pots" = "" ]; then
 	echo "No POT templates found in: $templates"
 	exit 1
 fi
diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/popuretext translate-toolkit-2.2.4/tools/popuretext
--- translate-toolkit-2.2.4.org/tools/popuretext	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/popuretext	2021-03-02 23:26:50.197044243 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Copyright 2005 Zuza Software Foundation
 #
@@ -27,7 +27,7 @@ if [ $# -lt 2 ]; then
 fi
 
 do_pot=0
-if [ "$1" == "-P" ]; then
+if [ "$1" = "-P" ]; then
 	do_pot=1
 	shift
 	potdir=$1
diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/poreencode translate-toolkit-2.2.4/tools/poreencode
--- translate-toolkit-2.2.4.org/tools/poreencode	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/poreencode	2021-03-02 23:26:50.197044243 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Copyright 2004 Zuza Software Foundation
 #
diff -urNp -x '*.orig' translate-toolkit-2.2.4.org/tools/posplit translate-toolkit-2.2.4/tools/posplit
--- translate-toolkit-2.2.4.org/tools/posplit	2017-07-31 15:41:45.000000000 +0200
+++ translate-toolkit-2.2.4/tools/posplit	2021-03-02 23:26:50.197044243 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Copyright 2004, 2007 Zuza Software Foundation
 #
